starwars/src/components/Card.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+
+const Card = props => {
+  return (
```

Comment:
consider object destructuring to dry up code. `const {name, birth_year, gender, height, mass, skin_color, hair_color, eye_color} = props.characterInfo`